test/org/apache/juli/TestFileHandlerNonRotatable.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.juli;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.apache.catalina.startup.LoggingBaseTest;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TestFileHandlerNonRotatable extends LoggingBaseTest {
+    private FileHandler testHandler;
+
+    @BeforeClass
+    public static void setUpPerTestClass() throws Exception {
+        System.setProperty("java.util.logging.manager",
+                "org.apache.juli.ClassLoaderLogManager");
+        System.setProperty("java.util.logging.config.file",
+                TestFileHandlerNonRotatable.class
+                        .getResource("logging-non-rotatable.properties")
+                        .getFile());
+    }
```

Comment:
added a line to decode URL path but this behavior is different in IDE (Eclipse in my case) and command line so please test both to be sure everything works